site/docs/master/start-contributing.md

```diff
@@ -9,6 +9,8 @@
 
 You may join the Velero community and contribute in many different ways, including helping us design or test new features. For any significant feature we consider adding, we start with a design document. You may find a list of currently in progress new designs here: https://github.com/vmware-tanzu/velero/pulls?q=is%3Aopen+is%3Apr+label%3ADesign. Feel free to review and help us with your input.
 
+You can also vote on issues using :+1: and :-1:, as explained in our [Feature enhancement request][3] and [Bug issue][4] templates. This will help us quantify importance and prioritize issues.
```

Comment:
I would make this same change in `site/docs/v1.4/start-contributing.md` as well, so it's immediately visible in the latest tagged docs.

Reply:
Adding!